src/tracker/services/torrent.rs

```diff
@@ -137,14 +137,16 @@ mod tests {
         use std::str::FromStr;
         use std::sync::Arc;
 
-        use crate::config::{ephemeral_configuration, Configuration};
+        use torrust_tracker_configuration::Configuration;
+        use torrust_tracker_test_helpers::configuration::ephemeral;
+
         use crate::protocol::info_hash::InfoHash;
         use crate::tracker::services::common::tracker_factory;
         use crate::tracker::services::torrent::tests::sample_peer;
         use crate::tracker::services::torrent::{get_torrent_info, Info};
 
         pub fn tracker_configuration() -> Arc<Configuration> {
-            Arc::new(ephemeral_configuration())
+            Arc::new(ephemeral())
```

Comment:
Hi @WarmBeer, In this cases, I like to use the super mod like this:

```rust
Arc::new(configuration::ephemeral())
````

Otherwise, it forces you to go up to read the "use" lines to know what `ephemeral` is? or to read what the function returns.